### ISSUE TITLE
Upload service is down

### ISSUE THREAD

**Issue by edsiper** [asciinema/asciinema-server#339]
troubleshooting the server response I get

```
$ python3 -m asciinema upload ../fluent-bit/asciinema.cast 
asciinema: upload failed: Error: 502 - 502 Bad Gateway

asciinema: retry later by running: asciinema upload ../fluent-bit/asciinema.cast
```

**Comment by sickill** [comment]
Seems to be working fine now. Were you able to upload?

**Comment by edsiper** [comment]
same error:

```
$ python3 -m asciinema upload ../fluent-bit/asciinema.cast 
asciinema: upload failed: Error: 502 - 502 Bad Gateway

asciinema: retry later by running: asciinema upload ../fluent-bit/asciinema.cast
```

**Comment by sickill** [comment]
Solved, see the comment in asciinema/asciinema#353.

**Comment by edsiper** [comment]
Thanks!
